code/drasil-docLang/Drasil/DocumentLanguage/TraceabilityGraph.hs

```diff
@@ -23,14 +23,14 @@ mkGraphInfo si = GI {
     , gsColour     = "darkgoldenrod1"
     , cColour      = "lavender"
 
-    , assumpLabels = (getLabels tvAssumps si,      "A")
-    , ddLabels     = (getLabels tvDataDefns si,    "DD")
-    , gdLabels     = (getLabels tvGenDefns si,     "GD")
-    , tmLabels     = (getLabels tvTheoryModels si, "TM")
-    , imLabels     = (getLabels tvInsModels si,    "IM")
-    , rLabels      = (getLabels tvReqs si,         "R")
-    , gsLabels     = (getLabels tvGoals si,        "GS")
-    , cLabels      = (getLabels tvChanges si,      "C")
+    , assumpLabels = (getLabels tvAssumps si,  "A")
```

Comment:
I can't help but think that these strings are already in `data-drasil` somewhere.